src/components/SingleDatePicker.jsx

```diff
@@ -128,6 +128,7 @@ class SingleDatePicker extends React.PureComponent {
       showKeyboardShortcuts: false,
```

Comment:
@monokrome I think the SingleDatePicker needs to be updated to match the DRP still. It's still using the `blur` event and it'd be good for the implementation to look as identical as possible with the class methods and so forth.

Reply:
@majapw Sure! I didn't do that part, so was getting everything right before the cleanup part of the code I wasn't working on directly. Working on it now 😺